huaweicloud/services/drs/resource_huaweicloud_drs_job.go

```diff
@@ -449,6 +450,7 @@ func resourceJobRead(_ context.Context, d *schema.ResourceData, meta interface{}
 		d.Set("multi_write", detail.MultiWrite),
 		d.Set("created_at", detail.CreateTime),
 		d.Set("status", detail.Status),
+		d.Set("tags", utils.TagsToMap(detail.Tags)),
```

Comment:
If we can fetch `tags` from API response, then we should update the `Import` section in docs and acceptance.

Reply:
The acceptance didn't add `tags` in `ImportStateVerifyIgnore` before. Docs is fixed now.